File: admin-web/typescript-ant-design-vue/src/api/business/data-tracer/model/data-tracer-vo.ts
/*
 * @Description: 
 * @version: 
 * @Author: lidoudou
 * @Date: 2021-08-27 17:52:02
 * @LastEditors: lidoudou
 * @LastEditTime: 2021-08-27 17:52:28
 */
export interface DataTracerVo { 
    /**
     * 单据id
     */
    businessId?: number;
    /**
     * 单据类型:  <br>  export const <br> DATA_TRACER_BUSINESS_TYPE_ENUM = <BR> {<br>&nbsp;&nbsp;CLUE_USER:{<br>&nbsp;&nbsp;&nbsp;&nbsp;value:1,<br>&nbsp;&nbsp;&nbsp;&nbsp;desc:\'线索\'<br>&nbsp;&nbsp;}<br>} <br>
     */
    businessType: number;
    /**
     * 单据类型描述
     */
    businessTypeDesc?: string;
    /**
     * 操作时间
     */
    createTime?: string;
    /**
     * 操作内容
     */
    operateContent?: string;
    /**
     * 操作类型
     */
    operateType?: number;
    /**
     * 操作类型描述
     */
    operateTypeDesc?: string;
    /**
     * 操作人
     */
    operatorId?: number;
    /**
     * 操作人名称
     */
    operatorName?: string;
}

File: admin-api/java-api/src/main/java/net/lab1024/smartadmin/module/support/datatracer/domain/DataTracerVO.java
package net.lab1024.smartadmin.module.support.datatracer.domain;

import io.swagger.annotations.ApiModelProperty;
import lombok.Data;
import net.lab1024.smartadmin.common.swagger.ApiModelPropertyEnum;
import net.lab1024.smartadmin.module.support.datatracer.constant.DataTracerBusinessTypeEnum;

import java.time.LocalDateTime;

/**
 * [  ]
 *
 * @author 罗伊
 */
@Data
public class DataTracerVO {

    @ApiModelProperty("单据id")
    private Long businessId;

    @ApiModelPropertyEnum(value = DataTracerBusinessTypeEnum.class, desc = "业务类型")
    private Integer businessType;

    @ApiModelProperty("单据类型描述")
    private String businessTypeDesc;

    @ApiModelProperty("操作类型")
    private Integer operateType;

    @ApiModelProperty("操作类型描述")
    private String operateTypeDesc;

    @ApiModelProperty("操作内容")
    private String operateContent;

    @ApiModelProperty("操作人")
    private Long operatorId;

    @ApiModelProperty("操作人名称")
    private String operatorName;

    @ApiModelProperty("操作时间")
    private LocalDateTime createTime;

}
